packages/lesswrong/components/common/Section.jsx

```diff
@@ -93,9 +93,12 @@ const Section = ({
   subscribeLinks = null,
   className = null,
   children,
-  classes
+  classes,
+  deactivateSection
 }) => {
 
+  if (deactivateSection) return <React.Fragment>{children}</React.Fragment>
```

Comment:
This also takes out an error boundary, which seems not ideal. Probably the right way to do this is to put `withErrorBoundary` on the component, and remove the `Components.ErrorBoundary` from the other branch.